[2023-07-16T15:27:33.652+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.install_libraries manual__2023-07-16T15:27:32.070374+00:00 [queued]>
[2023-07-16T15:27:33.662+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.install_libraries manual__2023-07-16T15:27:32.070374+00:00 [queued]>
[2023-07-16T15:27:33.662+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-16T15:27:33.678+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): install_libraries> on 2023-07-16 15:27:32.070374+00:00
[2023-07-16T15:27:33.684+0000] {standard_task_runner.py:57} INFO - Started process 1180 to run task
[2023-07-16T15:27:33.687+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'install_libraries', 'manual__2023-07-16T15:27:32.070374+00:00', '--job-id', '556', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpujl80oz_']
[2023-07-16T15:27:33.689+0000] {standard_task_runner.py:85} INFO - Job 556: Subtask install_libraries
[2023-07-16T15:27:33.733+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.install_libraries manual__2023-07-16T15:27:32.070374+00:00 [running]> on host 1696d5f40b5e
[2023-07-16T15:27:33.825+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='install_libraries' AIRFLOW_CTX_EXECUTION_DATE='2023-07-16T15:27:32.070374+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-16T15:27:32.070374+00:00'
[2023-07-16T15:27:33.826+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-16T15:27:33.827+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install networkx matplotlib scipy']
[2023-07-16T15:27:33.838+0000] {subprocess.py:86} INFO - Output:
[2023-07-16T15:27:34.661+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-07-16T15:27:34.862+0000] {subprocess.py:93} INFO - Requirement already satisfied: networkx in /home/***/.local/lib/python3.7/site-packages (2.6.3)
[2023-07-16T15:27:34.863+0000] {subprocess.py:93} INFO - Requirement already satisfied: matplotlib in /home/***/.local/lib/python3.7/site-packages (3.5.3)
[2023-07-16T15:27:34.863+0000] {subprocess.py:93} INFO - Requirement already satisfied: scipy in /home/***/.local/lib/python3.7/site-packages (1.7.3)
[2023-07-16T15:27:34.914+0000] {subprocess.py:93} INFO - Requirement already satisfied: cycler>=0.10 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (0.11.0)
[2023-07-16T15:27:34.916+0000] {subprocess.py:93} INFO - Requirement already satisfied: fonttools>=4.22.0 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (4.38.0)
[2023-07-16T15:27:34.917+0000] {subprocess.py:93} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (1.4.4)
[2023-07-16T15:27:34.919+0000] {subprocess.py:93} INFO - Requirement already satisfied: numpy>=1.17 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (1.21.6)
[2023-07-16T15:27:34.920+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging>=20.0 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (21.3)
[2023-07-16T15:27:34.921+0000] {subprocess.py:93} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (9.5.0)
[2023-07-16T15:27:34.923+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyparsing>=2.2.1 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (3.1.0)
[2023-07-16T15:27:34.924+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-dateutil>=2.7 in /home/***/.local/lib/python3.7/site-packages (from matplotlib) (2.8.2)
[2023-07-16T15:27:35.001+0000] {subprocess.py:93} INFO - Requirement already satisfied: typing-extensions in /home/***/.local/lib/python3.7/site-packages (from kiwisolver>=1.0.1->matplotlib) (4.7.1)
[2023-07-16T15:27:35.035+0000] {subprocess.py:93} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.7/site-packages (from python-dateutil>=2.7->matplotlib) (1.16.0)
[2023-07-16T15:27:40.704+0000] {subprocess.py:93} INFO - 
[2023-07-16T15:27:40.704+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.1.2 -> 23.2
[2023-07-16T15:27:40.705+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-07-16T15:27:40.949+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-16T15:27:40.978+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=install_libraries, execution_date=20230716T152732, start_date=20230716T152733, end_date=20230716T152740
[2023-07-16T15:27:41.012+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-16T15:27:41.061+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
